lib/sandbox/vault.js

```diff
@@ -33,11 +33,11 @@ class Vault {
 const getVaultInterface = (vault) => {
     return {
         get: (key) => {
-            return vault('get', key);
+            return vault('get', key).then((value) => { return (value === null ? undefined : value); });
         },
 
         set: (key, value) => {
-            return vault('set', key, value);
+            return vault('set', key, value).then(() => { return value; });
```

Comment:
Why are we making this change?

`.set` returns undefined for other variable scopes.